src/Common/CustomTagSelector/Types.ts

```diff
@@ -0,0 +1,63 @@
+import { KEY_VALUE } from '../Constants'
+import { OptionType } from '../Types'
+
+export interface TagType {
```

Comment:
Please rename the Types to type.ts or can specify as tagSelector.type.ts

Reply:
It will be Types.ts